hermes-console/static/partials/modal/editSubscription.html

```diff
@@ -177,11 +177,12 @@ <h3 class="modal-title" ng-show="operation === 'EDIT'"><small>Edit:</small> {{su
             <div class="form-group">
                 <label class="col-md-3 control-label">Filter</label>
                 <div class="col-md-9">
-                    <table class="table borderless" ng-show="subscription.filters.length > 0">
+                    <table class="table borderless">
                         <tr>
                             <th>Type</th>
                             <th>Path</th>
                             <th>Matcher</th>
+                            <th>#</th>
```

Comment:
can we leave an empty header cell `<th></th>`?
`#` is usually used for rows enumeration.

Reply:
of course, no problem.

Reply:
thanks :]